### DISCUSSION TITLE
Rustdesk on Oracle Linux 8

### DISCUSSION THREAD

**Discussion by ayubwaheed** [rustdesk/rustdesk#3178] [Q&A]
Hello,
I am trying to install rust desk on oracle linux8 where as its working fine on my other ubuntu vm. please assist experts.

Thanks

**Comment by Magvegva** [comment]
Hello! I make some minor changes in the installation script and it's works well on Oracle Linux 9.2
![изображение](https://github.com/rustdesk/rustdesk/assets/6399062/9702b0dd-66cd-4224-b02d-753d354533bd)

this line:

elif [ "$OS" = "CentOS" ] || [ "$OS" = "RedHat" ]  || [ "$OS" = "Oracle Linux Server" ]   || [ "${UPSTREAM_ID}" = "rhel" ] ; then

**Reply by rustdesk** [reply]
Thanks， PR please

**Reply by Magvegva** [reply]
Feel free to do it too)
